### PR TITLE
Thumbor compliance

### PR DESCRIPTION
**Pull request by giovanirinaldi**
- Removing (un)quotes, en/de/codes, and matching url regex after tornado so it doesnt encode in utf8 (i.e. respect/don't alter client's url)
- Passing through original headers, except for host. Added an option for forwarding it, if needed. (i.e. respect custom headers and forward host header to origin)

### CODE REVIEW

**Comment by dbiazus** [comment]
👍

**Comment by marcelomd** [comment]
🍰

File: thumbor/handlers/imaging.py [hunk 1]
```diff
@@ -34,12 +34,13 @@ def check_image(self, kw):
 
         url = self.request.path
 
-        kw['image'] = quote(kw['image'].encode('utf-8'))
+        #kw['image'] = quote(kw['image'].encode('utf-8'))
```
Comment (victorpoluceno) [inline thumbor/handlers/imaging.py]: Hey, its usually a bad idea to keep comment code, git is your friend if you need to revert this.

**Comment by victorpoluceno** [comment]
Shouldn't this change some of thumbor tests?

**Comment by victorpoluceno** [comment]
One more tip. I don't known about Thumbor contributing guidelines, I didn't find any, but usually maintainers will always ask you to split things on different commits. So, if we have any hope of contributing this back to Thumbor, you should make each feature one commit, that will probably make our life easier in the future.

**Comment by victorpoluceno** [comment]
@giovanirinaldi Just a few tips, few free to ignore any those if they don't make any sense.

**Comment by giovanirinaldi** [comment]
@victorpoluceno removed the comment code, replaced it for real comments about the change, thanks! About the test suite, we plan to integrate it into Jenkins for sure in a near future (and new tests regarding this changes will be needed). Finally, about contributing, we do not intend to, mainly due to the fact that these changes were either already suggested and/or pull-requested, but eventually rejected by thumbor's maintainers.

**Pull request merged**